dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -131,74 +133,133 @@ const searchBar = css`
 	padding-bottom: 15px;
 `;
 
+const editionList: EditionLinkType[] = [
+	{
+		url: '/preference/edition/int',
+		editionId: 'INT',
+		longTitle: 'International edition',
+		title: 'International edition',
+	},
+	{
+		url: '/preference/edition/au',
+		editionId: 'UK',
+		longTitle: 'UK edition',
+		title: 'UK edition',
+	},
+	{
+		url: '/preference/edition/us',
+		editionId: 'US',
+		longTitle: 'US edition',
+		title: 'US edition',
+	},
+	{
+		url: '/preference/edition/au',
+		editionId: 'AU',
+		longTitle: 'Australia edition',
+		title: 'AU edition',
+	},
+];
+
+const getEdition = (editionId: EditionId): EditionLinkType => {
+	return (
+		editionList.find((edition) => edition.editionId === editionId) ??
+		editionList[1]
+	);
+};
```

Comment:
I think a lot of this is actually defined in [editions.ts](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/lib/edition.ts) - could that code be reused & upgraded to achieve this?